pilot/pkg/model/gateway.go

```diff
@@ -79,13 +93,13 @@ func MergeGateways(gateways ...Config) *MergedGateway {
 				if p, exists := plaintextServers[s.Port.Number]; exists {
 					currentProto := ParseProtocol(p[0].Port.Protocol)
 					if currentProto != protocol || !protocol.IsHTTP() {
-						log.Debugf("skipping server on gateway %s port %s.%d.%s: conflict with existing server %s.%d.%s",
+						log.Warnf("skipping server on gateway %s port %s.%d.%s: conflict with existing server %s.%d.%s",
```

Comment:
The general approach is to add a metric - users are not expected to read the warnings.

Reply:
Don’t change this. This gets extremely spammy as it’s called often.